presidio-analyzer/README.MD

```diff
@@ -1,26 +1,23 @@
 # Presidio analyzer
 
 ## Description
+
 The Presidio analyzer is a Python based service for detecting PII entities in text.
 
-During analysis, it runs a set of different *PII Recognizers*, 
+During analysis, it runs a set of different *PII Recognizers*,
 each one in charge of detecting one or more PII entities using different mechanisms.
 
-Presidio analyzer comes with a set of predefined recognizers, 
+Presidio analyzer comes with a set of predefined recognizers,
 but can easily be extended with other types of [custom recognizers](#customizing-presidio-analyzer).
-Predefined and custom recognizers leverage regex, [spaCy](https://spacy.io/) 
-and other types of logic to detect PII in unstructured text.
+Predefined and custom recognizers leverage regex, [spaCy](https://spacy.io/) and other types of logic to detect PII in unstructured text.
   
 ## Installation
 
-To get started with Presidio-analyzer, 
-download the package and the `en_core_web_lg` spaCy model, 
-preferably in a virtual environment like Conda.
-
+To get started with Presidio-analyzer, download the package and the `en_core_web_lg` spaCy model, preferably in a virtual environment like Conda.
 
 ```sh
 pip install presidio-analyzer
-python -m spacy download en_core_web_sm
+python -m spacy download en_core_web_lg
```

Comment:
maybe 
SPACY_MODEL=<spacy model e.g. en_core_web_lg>
python -m spacy download $SPACY_MODEL

Reply:
Do you think this would make the doc more readable? We're not going to use the SPACY_MODEL param in other parts of this README.

Reply:
I like "for dummies" readmes, but this is nit picking.

Reply:
I would leave it as is because this is how spaCy proposes to use it in their docs. See here for example: https://spacy.io/usage/models